lua/rip-substitute/popup-win.lua

```diff
@@ -215,9 +215,18 @@ local function setPopupTitle()
 	if state.range then
 		title = "Range: " .. state.range.start
 		if state.range.start ~= state.range.end_ then title = title .. " – " .. state.range.end_ end
-		if state.useFixedStrings then title = title .. " [Fixed Strings]" end
-	elseif state.useFixedStrings then
-		title = "[Fixed Strings]"
+	end
+
+	if state.useFixedStrings then
+		title = title .. " 󰑑 󰅙"
```

Comment:
While I get a nerdfont is useful too prevent the title from getting too long, not everyone uses nerdfonts. And as opposed to the window title, this one is problematic, as it's undistinguishable from the case sensitive icon for people not using nerdfont. 

I propose to use the short form flags from ripgrep instead.

Reply:
It's worth a thought, but for this PR, I'd just go with the short form flags for now